frontend/assetic/yuicompressor.rst

```diff
@@ -14,7 +14,7 @@ How to Minify JavaScripts and Stylesheets with YUI Compressor
 
 Yahoo! provides an excellent utility for minifying JavaScripts and stylesheets
 so they travel over the wire faster, the `YUI Compressor`_. Thanks to Assetic,
-you can take advantage of this tool very easily.
```

Comment:
🤣 I can tell you it was a pain to get this thing to work, and even then it didn't work as expected!
This indicates the importance of a welcoming language 👍 great job!